provider/ansible.py

```diff
@@ -149,6 +150,18 @@ def distro_install():
         """
         Install ansible from the distro
         """
+        # Provide custom dnf repo containing ansible
+        if "ansible_repo" in params and params["ansible_repo"] != "":
```

Comment:
`if params.get('ansible_repo'):` can achieve the same thing :)

Reply:
Thanks :slightly_smiling_face: (Changed.)